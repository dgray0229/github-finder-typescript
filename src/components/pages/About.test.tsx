import React from 'react';
import { unmountComponentAtNode } from 'react-dom';
import { render } from '@testing-library/react';
import renderer from "react-test-renderer";
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


import About from './About';

Enzyme.configure({ adapter: new Adapter() })


describe("About Page", () => {
    test('renders successfully', () => {
        const wrapper = shallow(<About />)
        expect(wrapper.exists()).toBe(true)
      });  
});