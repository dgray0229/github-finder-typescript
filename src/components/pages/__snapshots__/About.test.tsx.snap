// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page renders successfully 1`] = `ShallowWrapper {}`;
